algorithms/unit_tests/TestBinSortA.hpp

```diff
@@ -219,6 +219,10 @@ void test_sort_integer_overflow() {
 }  // namespace BinSortSetA
 
 TEST(TEST_CATEGORY, BinSortGenericTests) {
+  // FIXME_OPENMPTARGET - causes ICE with CrayClang compiler
```

Comment:
These comment look wrong to me.  How are these skipping the test help if that was an internal compiler error?
Is that right?  The error was at compile time, not runtime?!